sql/sql.y

```diff
@@ -135,7 +135,7 @@
 %token <val> IDENT NUMBER STRING
 
 %left <val> AND OR
-%left <val> '>' '<' '=' GE LE
+%left <val> '>' '<' '=' '!' GE LE NE
```

Comment:
Since `NE` is supported, do we need to support `EQ`?

Reply:
EQ is supported by line 269: `| expr '=' expr  { $$ = binary('=', $1, $2, $3) }`. We may add `EQ` when we need it?